cmd/root.go

```diff
@@ -11,7 +13,8 @@ import (
 
 func newRootCmd() *cobra.Command {
 	options := lefthook.Options{
-		Fs: afero.NewOsFs(),
+		Fs:         afero.NewOsFs(),
+		ConfigPath: os.Getenv("LEFTHOOK_CONFIG"),
```

Comment:
Let’s move `os.Getenv` to the `load.go`? This way we won’d have to change function signature. For testing you can use `t.Setenv`